tests/testthat/test-utils.R

```diff
@@ -54,3 +54,117 @@ test_that("getSelector", {
   expect_equal(getSelector("mykey", "%", exact = TRUE), " AND mykey LIKE '%'")
   expect_equal(getSelector("mykey", "%", exact = FALSE), " AND mykey LIKE '%'")
 })
+
+test_that("configure_spark() returns early on windows", {
+  local_mocked_bindings(is_macos = function() {FALSE})
+
+  res <- configure_spark()
+
+  expect_equal(res, NULL)
```

Comment:
```suggestion
  expect_equal(configure_spark(), NULL)
```

Because this test isn't that important, I'd keep it as compact as possible.